wallet-new/integration/CLI.hs

```diff
@@ -62,5 +83,8 @@ data CLOptions = CLOptions
     , tlsPrivKeyPath    :: FilePath
     , tlsCACertPath     :: FilePath
     , serverHost        :: String
-    , serverPort        :: Int
+    , serverPort0       :: Int
```

Comment:
Would be nice to have a comment on why we need 4 separate ports.

Reply:
The idea is to have integration tests that spawn several nodes.
For example a transaction from wallet1 on node1 to wallet2 on node2.

Reply:
Ok, that seems totally reasonable, but it should be in a comment :)